spec/utility/circular_buffer_spec.rb

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe Bugsnag::Utility::CircularBuffer do
+  describe "buffer size" do
```

Comment:
For unit tests, I prefer to have the 2nd level `describe`s use the method names as the doc strings (e.g. this would be `describe '#max_items'`, lines 19-24 would be in its own `describe '#max_items='`)

Reply:
Personally I prefer tests to be based around expectations of functionality rather than necessarily method testing, as I find `the buffer size can be changed` is more useful when reading the tests than `max_items=`.  However, this is mostly semantics and I don't mind changing the test focus if it helps others read them.

Reply:
I'll restructure around methods, as I agree it may be preferable.